### PR TITLE
Update to Python 3.8, use slim image Fixes #191

### LINKED ISSUE #191: Change Python 3.7 image to Python 3.7-slim (or 3.8)
I believe that this script only really needs the Python 3.7 slim image.

This should save about 10GB in size on each image.

This is something I'm looking to try to do for all images.

Also can we use 3.8 as 3.7 will be out of support soon.

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # FROM directive instructing base image to build upon
-FROM python:3.7
+FROM python:3.8-slim
 
 RUN apt-get update && apt-get --no-install-recommends install --yes \
```
[the remark this answers is not in the document]
Comment (ssciolla) [reply Dockerfile:4]: I don't know all of these, but I'm pretty sure we're not using `netcat` or `git`(?).

**Comment by jonespm** [comment]
Yeah, I was just checking with you, I think we can slim this down more and just take care of it here. I can do that.

**Comment by jonespm** [comment]
It needs the default-libmysqlclient-dev and build-essential to build mysqlclient (Python). 

It needs git because requirements has a git+https and it does a
git clone of api-utils-python. We could release api-utils to pypi then we don't need git. But maybe that's not top priority right now. 

Removing everything but these 2 saves 40MB and about 15 packages. Removing git would get a little bit more but not much.

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonespm) [reply Dockerfile:4]: Yup, I cleaned them to the current essentials.

[the remark this answers is not in the document]
Comment (ssciolla) [reply Dockerfile:4]: Cool, thanks!

**Comment by jonespm** [comment]
I'm trying to see if I can get the api-utils released but I guess we can address this separately and just get this in.

**Pull request merged**